xwiki-platform-core/xwiki-platform-icon/xwiki-platform-icon-fontawesome/src/main/resources/IconThemes/FontAwesome.xml

```diff
@@ -37,370 +37,370 @@
   <syntaxId>plain/1.0</syntaxId>
   <hidden>true</hidden>
   <content>## General settings
-xwiki.iconset.type = font
-xwiki.iconset.ssx = IconThemes.FontAwesome
-xwiki.iconset.render.wiki = {{html clean="false"}}&lt;span class="fa fa-$icon"&gt;&lt;/span&gt;{{/html}}
-xwiki.iconset.render.html = &lt;span class="fa fa-$icon"&gt;&lt;/span&gt;
-xwiki.iconset.icon.cssClass = fa fa-$icon
+    xwiki.iconset.type = font
```

Comment:
The new spaces in front of every line look wrong to me, is this really a good idea? Won't this space end up in the page content?

Reply:
:+1: Even if this did not change page content, we should avoid changing the codestyle of the whole file when updating just a line. I should have checked the diff properly before opening the PR.
I fixed it in https://github.com/xwiki/xwiki-platform/pull/2541/commits/5cb34b044d7925768a37134d8f13ea30b8495dab and https://github.com/xwiki/xwiki-platform/pull/2541/commits/7894f798ff141f7113aeee52a1345c2a711e1b33
The diff looks clean now. :)